File: 2924-find-champion-ii/2924-find-champion-ii.js
var findChampion = function(n, edges) {
    const visited = Array(n).fill(false);
    for (const [u, v] of edges) {
        visited[v] = true;
    }
    
    let champion = -1;
    for (let i = 0; i < n; i++) {
        if (!visited[i]) {
            if (champion === -1) {
                champion = i;
            } else {
                return -1;
            }
        }
    }
    
    return champion;
};

File: 2924-find-champion-ii/2924-find-champion-ii.cpp
class Solution {
public:
    int findChampion(int n, vector<vector<int>>& edges) {
        bitset<100>v;
        for(auto it : edges)v[it[1]] = 1;
        int champion = -1;
        for(int i = 0 ; i < n ;i++){
            if(!v[i]){
                if(champion == -1)
                    champion = i; // 0
                else 
                    return -1;
            }
        }
        return champion;
        
    }
};